### DISCUSSION TITLE
I made improvements to the translation of simplified Chinese

### DISCUSSION THREAD

**Discussion by dzl333** [character-map-uwp/Character-Map-UWP#311] [General]
Here is the xlf:
[CharacterMap.zh-CN.zip](https://github.com/character-map-uwp/Character-Map-UWP/files/14125480/CharacterMap.zh-CN.zip)

(Please forgive possible grammatical mistakes, English is not my native language.)

**Comment by EdiWang** [comment]
Thank you, I will integrate your translation file shortly.

**Comment by EdiWang** [comment]
One thing, for `AppName`, I changed it back to `字符映射表`, without `UWP`, due to Microsoft Store naming reservation settings.

**Comment by EdiWang** [comment]
The changes are live now in version 2024.1.1. Thanks again for the help.
